<launch>
    <param name="num_robots" value="4" type="int"/>

    <arg name="launch_path" value="$(find modquad_simulator)/launch/aux/"/>


    <include file="$(arg launch_path)/modquad.launch">
        <arg name="model" value="modquad01"/>

        <arg name="init_x" default="0.5"/>
        <arg name="init_y" default="0.5"/>


        <arg name="color_r" default="1.0"/>
        <arg name="color_g" default="0.0"/>
        <arg name="color_b" default="0.0"/>
    </include>



    <include file="$(arg launch_path)/modquad.launch">
        <arg name="model" value="modquad02"/>

        <arg name="init_x" default="0.5"/>
        <arg name="init_y" default="0.5"/>


        <arg name="color_r" default="0.0"/>
        <arg name="color_g" default="1.0"/>
        <arg name="color_b" default="0.0"/>
    </include>





    <param name="num_robots" value="1" type="int"/>
    <!--Robot Visualization-->
    <!--<node pkg="modquad_simulator" type="visualizer.py" name="visual"/>-->
    <node name="rviz" pkg="rviz" type="rviz" args="-d $(find modquad_simulator)/rviz/four-robots.rviz" required="true"/>


    <!--<node pkg="dock_manager" type="docking_detector.py" name="docking_det"/>-->
    <!--<node pkg="modquad_simulator" type="docking_sim.py" name="docking_sim"/>-->
    <!--<node pkg="dock_manager" type="odom_multiplexer.py" name="odom_mux"/>-->


    <!--<node pkg="tf2_ros" type="static_transform_publisher" name="link1_broadcaster"-->
          <!--args="1 0 0 0 0 0 1 world link1"/>-->

</launch>
